pam_mysql.c

```diff
@@ -137,6 +137,8 @@
 #include <assert.h>
 #endif
 
+#include "compat_323_password.h"
```

Comment:
My only minor concern is that this include directive might need to be inside one of the conditionals to avoid duplicate symbols with versions where the mysql client library *does* contain the functionality, but I guess it's fine here.